primitives/consensus/common/src/import_queue.rs

```diff
@@ -181,11 +185,22 @@ pub enum BlockImportError {
 }
 
 /// Single block import function.
-pub fn import_single_block<B: BlockT, V: Verifier<B>, Transaction>(
+pub(crate) fn import_single_block_unmetered<B: BlockT, V: Verifier<B>, Transaction>(
```

Comment:
```suggestion
#[cfg(test)]
pub(crate) fn import_single_block_unmetered<B: BlockT, V: Verifier<B>, Transaction>(
```

Reply:
Unfortunately it is used in another crate and cannot be `#[cfg(test)]`